### PR TITLE
Switch to Meson

### PR DESCRIPTION
**Pull request by ernestask**
This PR moves the build system from Autotools to Meson, with tito for release management/RPM building.

Things will be painful initially, since a lot depends on immediately tagging a release, so we can actually use tito for things (1.2.9 and earlier will not be buildable per se). The laundry list is as follows:

- Convince Jenkins to build these
- Look into writing a custom tito tagger, so that the version is meson.build is bumped as well, otherwise write a script for that
- Look into a slight refactoring to allow passing a location to the resource bundle for a slightly less annoying development workflow (even though getting an RPM will be faster than before)
- ???

Testing is as easy as following the updated README. Some caveats:
- Faking a tito tag will be necessary (poke `.tito/packages/gnome-abrt` and add `1.2.9-1 ./`)
- The `run` run target is expectedly broken, but everything is mostly in place for that (had to copy all source files from srcdir to builddir, which is rather unorthodox, but ’tis Python)

### CODE REVIEW

**Comment by ernestask** [comment]
Eh, I should also hook pylint up.

**Comment by xsuchy** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -20,29 +20,23 @@ gnome-abrt is part of [the ABRT project](https://github.com/abrt/).
...
+    $ meson build
+    $ ninja -C build
 
 ### Checking
 
 gnome-abrt uses pylint to validate source codes. If pylint prints out any issue,
 the test will fail. Run the test by:
 
-    $ make check
+    $ ninja -C build test
```
Comment (mkutlak) [inline README.md]:
```
[1/2] Running all tests.
No tests defined.
```

There is also still a `make check` in spec file, and`checkoption` that en(dis)abled pylint.

Comment (ernestask) [reply README.md]: 1. Oops.
2. Linting was supposed to be in the description, but I forgot about it while I was typing, so thanks for the reminder.

Comment (ernestask) [reply README.md]: It’s there and it seems like it’s working.

**Comment by ernestask** [comment]
Added a custom tito tagger. The commit messages produced are a bit unwieldy, but a lot of it is hardcoded. Don’t want to copy over the entire implementation.

**Pull request merged**